### DISCUSSION TITLE
owncast LXC idea! ;3

### DISCUSSION THREAD

**Discussion by ScienceWolf666** [tteck/Proxmox#1986] [Ideas]
owncast LXC script idea ;3
https://owncast.online/docs/resources-requirements/

**Comment by tteck** [comment]
Ready for testing
```
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/owncast.sh)"
```

**Reply by ScienceWolf666** [reply]
Thank you very much, brooo hyhy ;3~ best regards and I'm about to start testing this now!
